### PR TITLE
Fixed permissions for GitHub action with welcome message.

### PR DESCRIPTION
**Pull request by felixxm**
Follow up to bc33b047848759e7ae90bf85c94ff6cc5913e1ee.

```
The workflow is not valid. .github/workflows/new_contributor_pr.yml (Line: 8, Col: 18): A sequence was not expected
```

Hopefully that's the final (3rd :exploding_head:) attempt. See [permissions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions) docs.

### CODE REVIEW

File: .github/workflows/new_contributor_pr.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     types: [opened]
 
 permissions:
-  pull-requests: [read, write]
+  pull-requests: read|write
```
Comment (ngnpope) [inline .github/workflows/new_contributor_pr.yml:8]: I'm thinking now that this ought to be:
```suggestion

**Review by ngnpope — COMMENTED** [review]
😆

File: .github/workflows/new_contributor_pr.yml (hunk 1 above)
Comment (felixxm) [reply .github/workflows/new_contributor_pr.yml]: Updated.

**Review by ngnpope — APPROVED** [review]
I feels a bit wrong for me to be approving this after two bungled attempts, but I guess this is harmless, the previous attempts didn't _look_ unreasonable, and GitHub's documentation is poor and ambiguous…

So ✔️ but feel free to wait for someone else if you'd prefer 🐱

Hopefully there won't be a 4th try!

**Pull request merged**